packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.js

```diff
@@ -1,13 +1,21 @@
 import { Box, Image } from 'grommet'
+import styled, { css } from 'styled-components'
+
 import getThumbnailSrc from '../../helpers/getThumbnailSrc.js'
 import { propTypes, defaultProps } from '../../helpers/mediaPropTypes.js'
 import useProgressiveImage from '../../../hooks/useProgressiveImage.js'
 
+const InlineBlock = styled(Box)`
+  display: inline-block;
+  ${props => props.height && css`max-height: ${props.height};`}
+  ${props => css`max-width: ${props.width || '100%'};`}
```

Comment:
I reviewed all storybooks on this branch and the changes are looking good so far. Before I move on to comparing FEM projects, I want to ask about handling of dimensions here.

On this branch, the Media story in lib-react-components shows an example of a Media component given only a custom height, and the behavior there is to crop the image. 
<img width="551" alt="production" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/e69069b4-d1fc-4030-991a-3e7141313da8">

However, on this branch, the height-only image looks layered behind the next image or might be overflowing its container:
<img width="554" alt="branch" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/3e6c9942-efec-40a5-a969-d4d14488dca1">

Is there a way to handle height-only Media components with the changes to ThumbnailImage in this branch?

Reply:
Another related note is that the placeholder image used in the Media story shows up differently here than production. For instance, on this branch the placeholder is smaller than its containing Media component:
<img width="406" alt="grey" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/2f215965-fc2e-4e42-8ed6-d58b0e93adc6">

But in production, the placeholder is teal and matches the dimensions of the Media component (200x200):
<img width="405" alt="teal" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/9e790b8c-a2f8-4f25-bfbd-b2a73fb5e51a">

I think we'd like to keep the placeholder the same size as the Media component when it's given a height and width - is that the correct assumption?

Reply:
I imagine we can handle those changes by reverting to flexbox. I didn't see any regressions on project pages, but I'm not sure where those might be using the placeholder, or height-only media.

Reply:
Yep, `inline-flex` fixes the storybook layout.

Reply:
The original changes here weren't passing through Grommet's style props to the `Box` component either. Hence the background colour vanishing, and so on. That's fixed now.